ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Image/ImagePage.tsx

```diff
@@ -85,6 +96,9 @@ function ImagePage() {
 
     const pagination = useURLPagination(DEFAULT_VM_PAGE_SIZE);
 
+    const hasGenerateSBOMAbility = useHasGenerateSBOMAbility();
+    const isScannerV4Enabled = isFeatureFlagEnabled('ROX_SCANNER_V4');
```

Comment:
Question for mental slow cooker: especially since `'ROX_SCANNER_V4'` is visible here, what are pro and con for hook to encapsulate the other requirements? Especially for RBAC questions (sometimes urgent to triage potential for it to be a problem on forum or support channels) I use humble Find in Files.

Reply:
I don't feel super strongly about this either, and can keep these explicit if it aids in debugging. The intention here was mostly for readability and avoiding repetition. 

I did encapsulate the `isFeatureFlagEnabled('ROX_SCANNER_V4')` call into a custom hook in a follow up PR though, which is moving more in the other direction. If having the feature flag/permission checks explicit at each use is preferable though I can revert.